File: app/src/main/java/com/karthik/myweather/network/model/LocationWeather.java
package com.karthik.myweather.network.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class LocationWeather implements Serializable {

    @SerializedName("id")
    public String id;

    @SerializedName("weather_state_name")
    public String weatherStateName;

    @SerializedName("weather_state_abbr")
    public String weatherStateAbbr;

    @SerializedName("min_temp")
    public double minTemperature;

    @SerializedName("max_temp")
    public double maxTemperature;

    @SerializedName("wind_speed")
    public double windSpeed;

    @SerializedName("applicable_date")
    public String date;
}

File: app/src/main/java/com/karthik/myweather/network/model/LocationWeather.kt
package com.karthik.myweather.network.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
data class LocationWeather(
        @SerializedName("id")
        var id: String? = null,

        @SerializedName("weather_state_name")
        var weatherStateName: String? = null,

        @SerializedName("weather_state_abbr")
        var weatherStateAbbr: String? = null,

        @SerializedName("min_temp")
        var minTemperature: Double = 0.0,

        @SerializedName("max_temp")
        var maxTemperature: Double = 0.0,

        @SerializedName("wind_speed")
        var windSpeed : Double= 0.0,

        @SerializedName("applicable_date")
        var date: String? = null
) : Parcelable